### PR TITLE
Use Unicorn on Heroku deployments

### PR DESCRIPTION
**Pull request by eoinkelly**
This PR completes the setup of Unicorn. There are some errors in Raygun related to some rails processes timing out waiting for a connection from the pool. I'm hoping this change will fix those too.

There is a review app for this change. I clicked around it and it seems to work fine. I will clean up the review app when this PR is closed.

### CODE REVIEW

**Comment by eoinkelly** [comment]
I think this is safe to go through to production but no worries if you would prefer to wait until the new year to merge it @Br3nda

**Review by Br3nda — APPROVED** [review]

**Comment by Br3nda** [comment]
All good. I'm happy as long as you're the one who will keep an eye on it in case it needs to be undone.

**Comment by eoinkelly** [comment]
Sweet, I'm happy to do that.

**Pull request merged**
